interface-definitions/service_pppoe-server.xml.in

```diff
@@ -47,6 +47,12 @@
                   </leafNode>
                 </children>
               </node>
+              <leafNode name="any-login">
```

Comment:
Why not call this `no-auth` like accell is doing?

Reply:
this is a different option.
from the documentation:
```
"any-login=" 0\|1
If set to 1 allows pass PAP, CHAP and Microsoft CHAP v1 authentication with any login.

"noauth=" 0\|1
If set to 1 skips any PPP and IPOE authentication.
```